pulseapi/profiles/serializers.py

```diff
@@ -12,3 +14,46 @@ class Meta:
         Meta class. Again: because
         """
         model = UserBookmarks
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    """
+    Serializes a user profile.
+    """
+    user_bio = serializers.CharField(
+        max_length=140,
+        required=False
+    )
+    custom_name = serializers.CharField(
+        max_length=500,
+        required=False
+    )
+    is_group = serializers.BooleanField()
+    thumbnail = serializers.ImageField(
+        max_length=2048,
+        required=False
+    )
+    issues = serializers.SlugRelatedField(
+        many=True,
+        slug_field='name',
+        queryset=Issue.objects,
+        required=True
+    )
+
+    twitter = serializers.models.URLField(max_length=2048)
```

Comment:
man I don't even know - this was copy-pasta and clearly django doesn't care to check whether this even works in the slightest.